src/streaming/models/CmcdModel.js

```diff
@@ -387,6 +387,9 @@ function CmcdModel() {
         if (mediaType === Constants.AUDIO) {
             ot = CmcdObjectType.AUDIO;
         }
+        if (request.mediaType === Constants.ENHANCEMENT) {
+            ot = 'e';
```

Comment:
Maybe not an issue, but where does this magic constant string come from? Is it allowed by Cmcd-spec? (@dsilhavy )

Reply:
Originally we have considered that the enhancement doesn't quite fit under existing Cmcd Object Types. However, after looking at the Cmcd-spec it seems that tokens not defined in the spec (eg. `e`) would be discarded by a Cmcd server.

I would like to assign the enhancement `ot` to `CmcdObjectType.OTHER`. Do you think this would be a good approach?